.github/workflows/publish.yml

```diff
@@ -25,16 +25,12 @@ jobs:
       - run: npm install
       - run: npm run build
       - run: npm run lint
-      - run: npm run coverage
+
+      # Test coverage is handled by tests.yml
+      - run: npm run test
         env:
           CONTINUOUS_INTEGRATION: 1
 
-      # Upload coverage data
-      - name: Coveralls
-        uses: coverallsapp/github-action@master
-        with:
-          github-token: ${{github.token}}
```

Comment:
Removing this ensures that a build with passing tests, but with a failure while uploading coverage data doesn’t prevent publishing of the package.